ext/RMagick/rmagick.h

```diff
@@ -1153,6 +1154,7 @@ extern VALUE  rm_enum_new(VALUE, VALUE, VALUE);
 // rmstruct.c
 extern VALUE  ChromaticityInfo_to_s(VALUE);
 extern VALUE  ChromaticityInfo_new(ChromaticityInfo *);
+extern void   Color_to_PixelColor(PixelColor *, VALUE);
 extern void   Color_to_PixelPacket(PixelPacket *, VALUE);
```

Comment:
I think `Color_to_PixelPacket()` should be removed. How do you think?

Reply:
We will still need this inside `Image_pixel_color` where the pixels of an image are accessed. We probably don't need this for ImageMagick7 but we need it for this particular case for now.